release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,15 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 5 May 2023 - Fix -  CloudGateway 2.10.10 - NullReferenceException breaking the cloud connection [ID_36346]
+
+Resolved an issue that was introduced since CloudGateway 2.10.8 where the connection to the could become broken. The CloudGateway log file would periodically contain the next exception:
```

Comment:
```suggestion
Resolved an issue that was introduced since CloudGateway 2.10.8 where the connection to the cloud could become broken. The CloudGateway log file would periodically contain the next exception:
```